tests/openvino/tools/calibrate.py

```diff
@@ -91,6 +94,12 @@ class ACValidationFunction:
     Implementation of a validation function using the Accuracy Checker.
     """
 
+    METRIC_TO_PERSAMPLE_METRIC = {
+        "coco_orig_precision": "coco_precision",
```

Comment:
Don't we need to match all of such kinds of metrics from POT into `calibrate.py`?
https://github.com/openvinotoolkit/openvino/blob/master/tools/pot/openvino/tools/pot/algorithms/quantization/accuracy_aware_common/utils.py#L28

Reply:
Done